src/components/RadioGroup/RadioGroupItem.tsx

```diff
@@ -172,7 +172,7 @@ class RadioGroupItem extends AutoControlledComponent<Extendable<IRadioGroupItemP
         onClick={this.handleClick}
         className={classes.root}
       >
-        <Label styles={styles.label}>
+        <Label styles={styles.content}>
```

Comment:
a bit worrying about semantics behind this nesting here. Do we really want the content (that is quite generic by its nature) to be always a children of `Label` (and, thus, to always reflect its styles and, what is more important, behavioural and accessibility aspects - for _any value of `content`_ that will be passed in)?

Reply:
I'm not changing the implementation as you can see, just the name of the prop (I should have kept the name of the styles slot to `label`, will revert that part).

I think this is a legitimate concern, but the nature of the radio button dictates to us that we need an `icon` with some `content` after it; maybe we should not use `Label` to wrap these, any suggestions?

@kuzhelov @mnajdova

Reply:
yes, I would suggest exactly the same path as you've done - specifically, to not wrap Icon with Label (especially if we would take into account where Icon is used - it is used to 'fill' the circle area of radio input). Then it would be much easier to decouple all these parts from each other.

Not sure if we should do it in this PR or create a dedicated one as a close follow-up, but the most important thing would be, actually, if we all agree on that proposal :)

Reply:
I'd rather do it in another PR; I'll discuss with @jurokapsiar as well to understand what are the concerns (if any) and proceed with removing Label